packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -253,6 +253,8 @@ const ToggleMenuDrawer = React.forwardRef(
         };
 
         const getLanguageRoutes = () => {
+            const currentLanguage = getLanguage();
```

Comment:
What difference does this make?
`currentLanguage` from utils file is pointing into `getLanguage` in /translations.

Reply:
While using the current language from utility folder we kept receiving the language that we logged in with and with the changes of the language our current language was not getting updated. I call getLanguage in our getLanguageRoutes function so that we're gonna have our current language updated with the changes of our language to highlight the current language which we did not have before by using the utils variable.